### ISSUE TITLE
OFE deployment fails when deploying from macOS.

### ISSUE THREAD

**Issue by ek-nag** [GoogleCloudPlatform/hpc-toolkit#978]
### Describe the bug

OFE deployment fails when deploying from macOS.

### Steps to reproduce

Steps to reproduce the behaviour:

1. clone develop branch of hpc-toolkit
2. run community/front-end/ofe/deploy.sh on macOS
3. deployment will fail.

### Expected behaviour

Successfully deployed OFE in GCP

### Actual behaviour

OFE VM startup script fails because OFE source code is put in the wrong directory.
startup script expect hpc-toolkit to be in `/opt/gcluster/`, however when deployed from macOS it never end up in there.
Certain commands used in `deploy.sh` are not compatible with macOS. For example:

macOS does not support `--recursive` flag used with `cp` command, or `rm` does not accept `--force --recursive`.

### Version (`ghpc --version`)
v1.13.0

### Output and logs

```text
Mar  3 11:06:52 mac-dep-server google_metadata_script_runner[1454]: startup-script-url: tar: Removing leading `../' from member names
Mar  3 11:06:52 mac-dep-server google_metadata_script_runner[1454]: startup-script-url: tar: ../._hpc-toolkit: Member name contains '..'
Mar  3 11:06:52 mac-dep-server google_metadata_script_runner[1454]: startup-script-url: tar: Ignoring unknown extended header keyword 'LIBARCHIVE.xattr.com.apple.provenance'
Mar  3 11:06:52 mac-dep-server google_metadata_script_runner[1454]: startup-script-url: tar: ../hpc-toolkit/: Member name contains '..'
```

### Execution environment

- OS: macOS
- Shell zsh
- go version 1.19.5

**Comment by ek-nag** [comment]
Resolved in #979

**Comment by tpdownes** [comment]
This fix is on our develop branch; we will close the issue when it's an officially tagged release
